frontend/packages/noobaa-storage-plugin/integration-tests/tests/objectServiceDashboard.scenario.ts

```diff
@@ -0,0 +1,214 @@
+  resiliencyOfMCG,
+  CapacityBreakdown,
+  PerformanceCard,
+  unifiedHealthButton,
+  POPOVER_GREEN_COLOR,
+  unifiedDataResiliencyButton,
+  mcgPopover,
+  rgwPopover,
+} from '../views/noobaaDashboardPage.view';
+import { testBucket } from '../mocks/obcData';
+import { RGW_PROVISIONER } from '../utils/consts';
+
+const loadAndWait = async () => {
+  await browser.get(`${appHost}/dashboards/object-service`);
+  await browser.wait(until.and(untilNoLoadersPresent));
+};
+
+const storageClasses = JSON.parse(execSync('oc get StorageClasses -o json').toString());
+const isRGWPresent = storageClasses.items.some((sc) => sc.provisioner === RGW_PROVISIONER);
+
+describe('Tests Buckets Card', () => {
+  beforeAll(async () => {
+    await loadAndWait();
+  });
+
+  it('Test at least one noobaa bucket is present', async () => {
+    await browser.wait(until.and(untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(noobaaCount));
+    const text = await noobaaCount.getText();
+    const [count] = text.split(' ');
+    expect(Number(count)).toBeGreaterThanOrEqual(1);
+  });
+
+  it('Create an Object Bucket Claim and test equality', async () => {
+    await browser.wait(until.visibilityOf(obcCount));
+    let text = await obcCount.getText();
+    const [initialCount] = text.split(' ');
+    execSync(`echo '${JSON.stringify(testBucket)}' | kubectl create -f -`);
+    await waitFor(obcCount, (Number(initialCount) + 1).toString());
+    text = await obcCount.getText();
+    const [finalCount] = text.split(' ');
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+    execSync(`echo '${JSON.stringify(testBucket)}' | kubectl delete -f -`);
+  });
+});
+
+if (!isRGWPresent) {
+  describe('Test Status Card', () => {
+    beforeAll(async () => {
+      await loadAndWait();
+    });
+
+    it('Check if Multi Cloud Gateway is in a healthy state', async () => {
+      await browser.wait(until.and(untilNoLoadersPresent));
+      await browser.wait(until.visibilityOf(healthOfMCG));
+      const color = await healthOfMCG.getAttribute('fill');
+      expect(color).toBe(OCP_HEALTH_ICON_COLORS.GREEN);
+    });
+
+    it('Check if Data Resiliency of MCG is in healthy state', async () => {
+      await browser.wait(until.visibilityOf(resiliencyOfMCG));
+      const color = await resiliencyOfMCG.getAttribute('fill');
+      expect(color).toBe(OCP_HEALTH_ICON_COLORS.GREEN);
+    });
+  });
+}
+
+describe('Test Object Data Reduction Card', () => {
+  beforeAll(async () => {
+    await loadAndWait();
+  });
+
+  it('Check if Efficiency Ratio is in acceptable data range', async () => {
+    await browser.wait(until.and(untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(efficiencyValue));
+    const effValue = await efficiencyValue.getText();
+    const [ratioA, ratioB] = effValue.split(':');
+    const [numA, numB] = [Number(ratioA), Number(ratioB)];
+    expect(numA).toBeGreaterThan(0);
+    expect(numB).toBeGreaterThan(0);
```

Comment:
If I am not wrong, efficiency ratio will always be like `x : 1` i.e. the numB will be 1 always and you should be checking for `numB === 1` here. @a2batic Can confirm.